src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConstants.cs

```diff
@@ -0,0 +1,266 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+using Microsoft.AspNetCore.Authentication;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Contains constants specific to the <see cref="SuperOfficeAuthenticationHandler"/>.
+    /// </summary>
+    public static class SuperOfficeAuthenticationConstants
+    {
+        public static class ClaimNames
+        {
+            /// <summary>
+            /// Tenant's user identity.
+            /// </summary>
+            public const string AssociateId = "http://schemes.superoffice.net/identity/associateid";
+
+            /// <summary>
+            /// Current user's company name.
+            /// </summary>
+            public const string CompanyName = "http://schemes.superoffice.net/identity/company_name";
+
+            /// <summary>
+            /// Current user tenant identifier.
+            /// </summary>
+            public const string ContextIdentifier = "http://schemes.superoffice.net/identity/ctx";
+
+            /// <summary>
+            /// Current SuperId user email address.
```

Comment:
Should this be _"SuperOffice Id"_, or is _"SuperId"_ some different terminology SuperOffice uses?

Reply:
This is SuperOffice terminology used for the STS.